src/app/modal/modal.controller.js

```diff
@@ -0,0 +1,229 @@
+        if ($uploadContainer) {
+            $uploadContainer.classList.toggle('valid', valid);
+        }
+    }
+
+    enableSendButton(isFormValid) {
+        var resume = this.ApplyService.form.resumeInfo;
+
+        if (isFormValid && (resume || this.linkedInData.resume)) {
+            if (this.linkedInData.resume.length !== 0 || resume.type) {
+                return false;
+            }
+        } else if (this.email) {
+            return false;
+        }
+        return true;
+    }
+
+    getTooltipText() {
+        var tooltip = '<ul>';
+        this.configuration.acceptedResumeTypes.forEach(function (type) {
+            tooltip += '<li>' + type + '</li>';
+        });
+        tooltip += '</ul>';
+        return tooltip;
+    }
+
+    formatResume(userProfile) {
+        var lineBreak = '\n',
+            hardBreak = '\n\n\n',
+            months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'June', 'July', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'],
+            today = new Date(),
+            friendlyDate = today.toLocaleDateString() + ' at ' + today.toLocaleTimeString(),
+            legal = 'This LinkedIn profile information was received on ' + friendlyDate + '. \n\nIt contains confidential information and is intended only for use within the Bullhorn platform as a part of the Career Portal app.\n\n';
+
+        // First Name
+        this.linkedInData.header = (userProfile.formattedName || '') + lineBreak;
+        // Email Address
+        this.linkedInData.header += (userProfile.emailAddress || '') + lineBreak;
+        // Location
+        if (userProfile.location && userProfile.location.name) {
+            this.linkedInData.header += (userProfile.location.name || '') + ', ';
+            if (userProfile.location.country) {
+                this.linkedInData.header += (userProfile.location.country.code.toUpperCase() || '') + hardBreak;
+            }
+        }
+        // Work Experience Block
+        this.linkedInData.resume = 'Work Experience:' + lineBreak;
+        // Positions
+        if (userProfile.positions) {
+            if (userProfile.positions.values && userProfile.positions.values.length) {
+                this.linkedInData.resume += (userProfile.positions.values[0].company.name || '') + ' ';
+                // Start Date
+                if (userProfile.positions.values[0].startDate) {
+                    this.linkedInData.resume += months[userProfile.positions.values[0].startDate.month - 1] + ' ' + userProfile.positions.values[0].startDate.year + ' - ' || '';
+                }
+                // End Date or 'Present'
+                if (userProfile.positions.values[0].endDate) {
+                    this.linkedInData.resume += months[userProfile.positions.values[0].endDate.month - 1] + ' ' + userProfile.positions.values[0].endDate.year || '';
+                } else {
+                    if (userProfile.positions.values[0].isCurrent) {
+                        this.linkedInData.resume += 'Present';
```

Comment:
So should all the dates.

K.

On Monday, December 21, 2015, Joshua Godi <notifications@github.com> wrote:

> In src/app/modal/modal.controller.js
> <https://github.com/bullhorn/career-portal/pull/27#discussion_r48215395>:
>
> > +        // Work Experience Block
> > +        this.linkedInData.resume = 'Work Experience:' + lineBreak;
> > +        // Positions
> > +        if (userProfile.positions) {
> > +            if (userProfile.positions.values && userProfile.positions.values.length) {
> > +                this.linkedInData.resume += (userProfile.positions.values[0].company.name || '') + ' ';
> > +                // Start Date
> > +                if (userProfile.positions.values[0].startDate) {
> > +                    this.linkedInData.resume += months[userProfile.positions.values[0].startDate.month - 1] + ' ' + userProfile.positions.values[0].startDate.year + ' - ' || '';
> > +                }
> > +                // End Date or 'Present'
> > +                if (userProfile.positions.values[0].endDate) {
> > +                    this.linkedInData.resume += months[userProfile.positions.values[0].endDate.month - 1] + ' ' + userProfile.positions.values[0].endDate.year || '';
> > +                } else {
> > +                    if (userProfile.positions.values[0].isCurrent) {
> > +                        this.linkedInData.resume += 'Present';
>
> Present should be localized.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bullhorn/career-portal/pull/27/files#r48215395>.
>